app/globals.css

```
@@ -65,39 +65,44 @@
 
   /* Typography - Font Weight */
   --font-weight-regular: 400;
+  --font-weight-medium: 500;
   --font-weight-semibold: 600;
   --font-weight-bold: 700;
+  --font-weight-extrabold: 800;
 
   /*layout */
```

Stylelint (17.2.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)